pkg/qemu/qemu.go

```diff
@@ -437,6 +465,10 @@ func Cmdline(cfg Config) (string, []string, error) {
 	} else if !isBaseDiskCDROM {
 		args = append(args, "-drive", fmt.Sprintf("file=%s,if=virtio", baseDisk))
 	}
+	for _, volDisk := range volumeDisks {
+		args = append(args, "-drive", fmt.Sprintf("file=%s,if=virtio", volDisk))
```

Comment:
Does this lock the file so that the same volume can't be written by multiple instances simultaneously?

Reply:
No, it doesn't. Just tested attaching a volume to two instances at once and writing to it from each of the instances - they overwrote each other and corrupted the volume. I'll introduce some sort of locking mechanism